### ISSUE TITLE
hooks not enable or disable at all.

### ISSUE THREAD

**Issue by hogo20** [PrestaShop/PrestaShop#35176]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

hello frineds
In prestashop 8.1.3, hooks  not enabled or disabled.
After I enable a hook (or disable) and refreshing the page I see that it is disabled (enabled) again.

### Expected behavior

It should be activated and remain in that state.

### Steps to reproduce

1. in prestashop 8.1.3, Go to "Designs" > "Positions"
2. for example search for "displayProductAdditionalInfo" and enable it.
3. in browser refresh the page,
4. I see it goes to "disable"

https://github.com/PrestaShop/PrestaShop/assets/11363195/d49e9a56-880c-4ce5-8294-b23cb6799f96

### PrestaShop version(s) where the bug happened

8.1.3

### PHP version(s) where the bug happened

7.4.33

**Comment by paulnoelcholot** [comment]
Hello @hogo20,

I reproduce the issue with PrestaShop version 8.1.x, I'll add this to the backlog so it can be fixed.

Please be aware that some issues might take a very long time to be resolved. If this one is important to you and you cannot wait for it to be fixed on the project’s own time, we strongly suggest you consider [contacting a professional](https://www.prestashop-project.org/support/) to help you. 

If you fix the issue on your end, please [contribute it back to the project](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.

Thank you

**Comment by hogo20** [comment]
Hello friends
I found a workaround to solve this problem. If someone has such a problem, they can temporarily use the following method.
Go to phpmyadmin in cpanel and click on the store database and find the hook table and enable or disable the desired hook.

**Comment by kpodemski** [comment]
@PrestaShop/qa-functional can somebody try to reproduce this issue again on PrestaShop 8.1? I didn't reproduce it and @florine2623 report is related to the `develop` branch...

**Comment by hogo20** [comment]
hello everyone
my  problem is a outdate module now II find "how to fix it".
1- I clicked "Disable non built-in modules" in "advanced parameters>performance".
 2- I went to the modules section and Enable  modules one by one, and after each module activation in the hooks section, I en/dis a hook to see if its Ajax window is displayed or not.
3- I found which non-original module caused this problem.

now my shop is okay. thank you very much
